dist2src/worker/updater.py

```diff
@@ -125,6 +125,12 @@ def _create_task(self, project: PagureProject, branch: str, commit: str):
             "end_commit": commit,
         }
         logger.debug(f"Sending task {task_name!r}, with payload: {event}")
-        r = celery_app.send_task(name=task_name, kwargs={"event": event})
-        logger.info(f"Task UUID={r.id} sent to Celery.")
+        r = celery_app.send_task(
+            name=task_name,
+            expires=self.cfg.update_task_expires,
+            kwargs={"event": event},
+        )
+        logger.info(
+            f"Task UUID={r.id}, expiring in {self.cfg.update_task_expires} seconds sent to Celery."
```

Comment:
Having "expiring in 82800 seconds" in each of these lines doesn't bring much value (to me).
What about:
```suggestion
            f"Task UUID={r.id}, expiring in at {datetime.now() + timedelta(seconds=self.cfg.update_task_expires)} sent to Celery."
```

Reply:
Good point, thanks!

Made an update to log this value when `check_updates` starts, next to the other configuration values. WDYT?